YogaKit/Source/YGLayout.m

```diff
@@ -89,7 +89,8 @@ - (void)set##objc_capitalized_name:(CGFloat)objc_lowercased_name
 
 @interface YGLayout ()
 
-@property (nonatomic, weak, readonly) UIView *view;
+@property (nonatomic, assign, readonly) YGNodeRef node;
```

Comment:
It does not affect the memory footprint, but it does reduce the number of msgSends. So probably not a bad idea.